Initial page for coaching teams (#7) (#13)
Programming-Contest-Foundation/pcf.community: content/wiki/Coaching-a-team.md

---
title: "Coaching a team"
date: 2022-12-02T11:50:27+02:00
---
In programming contests a team has a coach. A coach registers a team for the contest and is the main contact between 

> the organisers of the contest and the team. There are other things a coach can do for a team. This article will

Reviewer: I would give this first sentence a headline of its own. So that if you look at the layout of the page you see a complete list of headings of what a coach can do, not only the "extra" things a coach can do if you know what I mean. This sentence could also (later) be expanded upon e.g. about what registering a team means.

Author: This is actually already done by the template engine. The title is always added as the first header. It is a bit confusing if you don't know this quirk of the template. 
![image](https://user-images.githubusercontent.com/5221158/205499897-6c7c5598-82e4-4f48-9371-9c843a00ec9f.png)

* [TU Delft template](https://commissies.ch.tudelft.nl/chipcie/resources/tcr-template.zip) ([pdf](https://commissies.ch.tudelft.nl/chipcie/resources/tcr.pdf))
* [Does god play Tetris](https://github.com/alexjbest/icpc-tcr/blob/master/teamref.pdf)
* [Team hacKIT](https://github.com/niklasb/tcr/blob/master/tcr.pdf)
* [Omogen Heap](https://github.com/kth-competitive-programming/kactl/blob/main/kactl.pdf)

The TRD should be in a folder, or at least be stapled together. Make sure the teams bring a printed version to the 
contest, since not all contests offer the opportunity to print the document on site.

Other tips you can give your teams as coach is to check the schedule and see when/if there is dinner. If the award
ceremony is before the dinner, advice your teams to bring some snacks for after the contest. 

Another common issue is the temperature in the contest halls. It can heat up very fast with many people and pc's at the 
same time. Dressing in easily removable layers can help manage the temperature. Note that the T-shirt should usually
be worn on top and make sure your contestants are aware of it.

> In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
> s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on per 

Reviewer 2: ```suggestion
In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this is known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on per 
```

> In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
> s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on per 

Reviewer 2: ```suggestion
In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on a per 
```

> In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this i
> s known by the organisation. In ICPC there is a special needs field the contestants should fill in. It depends on per 

Reviewer 2: ```suggestion
In case one of your team members has dietary restrictions (vegetarian, vegan, gluten free, nut allergy), make sure this is known by the organization. In ICPC there is a special needs field the contestants should fill in. It depends per 
```

> contest how this is handled. Making sure there is food that fits the needs of the contestant always greatly helps 
> their results.

Reviewer 2: ```suggestion
contest how this is handled. Making sure there is food that fits the needs of the contestant, as this greatly helps their results.
```

Also check the systems page if all expected IDEs are present. Sometimes teams prefer a specific IDE that is not listed.
Contacting the organisation beforehand has a better chance of it being available then during the contest.

# Travelling to the contest
An important part the coach can play is arranging funding for travel. Often the university is willing to support by 

> financing (a part) of the travel expenses of the teams. 

Reviewer 3: ```suggestion
financing (a part of) the travel expenses of the teams. 
```
